### DISCUSSION TITLE
Show the picture while viewing searched items

### DISCUSSION THREAD

**Discussion by Jim-Young** [TWiStErRob/net.twisterrob.inventory#355] [Questions]
You want to see the picture of the item to make sure it's the one you are after. But it only displays the description of the item and where it's located. It also displays the pictures of the contents inside this item.

![Screenshot_2023-10-10-17-00-27](https://github.com/TWiStErRob/net.twisterrob.inventory/assets/29013899/8bcfe51a-55fb-447d-ab54-5b0cc5da0c22)

**Comment by TWiStErRob** [comment]
You're opening the item from search results. You have two options:
 * swipe sideways on the details to see the image
 * in Settings, change "Default Details Page" to "Image"

The idea behind this is that the search results shows images, so you've already seen what you're opening, and it opens the details page so that you can verify it's the one you're after (in the right room/box); or know where it is, because you're searching for it and that assumes you want to grab it.

**Reply by Jim-Young** [reply]
The searching list doesn't actually show the image and when you press on one of the result, it leads to the description(details page). Swiping sideways does show the image and solve the problem.

It would be even better if there is a hint saying how to show the image.

![Screenshot_2023-10-17-10-52-26](https://github.com/TWiStErRob/net.twisterrob.inventory/assets/29013899/3765c257-ba49-4682-a27c-5768430ba5c0)
